### ISSUE TITLE
Add to-do list to kat

### ISSUE THREAD

**Issue by noamblitz** [minvws/nl-kat-coordination#1665]

**Comment by underdarknl** [comment]
Agree, I think the first 'design' would be to add the 10 newest open questions on the crisis-room / dashboard, and add a link to all open questions in the top navigation.
The notificaton system can be more or less general, based on BIT like triggers, which il do a writeup of.

**Comment by noamblitz** [comment]
As discussed in standup, we need ways of knowing when a Question is answered. Some solutions:

1. Add a boolean field to the Question OOI. This will work correctly, but you lose some context -> what was the answer?
2. Add the resulting config as an attribute to the Question. This resolves the context problem but loses flexibility in the Question OOI -> it is then expected that all Questions result in a Config.

**Comment by underdarknl** [comment]
@noamblitz could we add an option 'answer' relation on the quesionOOI, which we populate with whatever OOI_ID (or Id's) was created after the answer was collected and parsed?

**Comment by noamblitz** [comment]
Yes, we could! Just like finding has an "ooi" attribute

**Comment by RomijnHumanoids** [comment]
21-01-2025 When working on #3890, the need for a 'to do list' got emphasised. Admins need to approve the export of new objects (e.g. for possible sensitive info to an external html endpoint). I've put this ticket in 'to do' for next sprint, 31 januari 2025.

This feature is also needed for admins to approve/delete proposed boefje variants #3334. Design inspiration could be taken from parts of this.

**Comment by underdarknl** [comment]
This can be implemented as a bunch Dashboards.
We could add default dashboards that have the following object queries:
* New Findings. (sorted on severity, since this week)
* Question OOI's that have no associated Config

**Comment by RomijnHumanoids** [comment]
First design reviewed in stakeholder meeting on 26 march 2025, and now for review with new changes: https://www.figma.com/design/IeJhr10MyKa2CL6qZSiNJy/%E2%9C%8D%F0%9F%8F%BC-KAT-%7C-For-Review-%7C-Dashboard?node-id=2644-53446&t=aRNPmXlMMufYeyux-1
